__tests__/app.test.js

```diff
@@ -41,15 +41,74 @@ describe('GET /api', () => {
         .get("/api")
         .expect(200)
         .then((res) => {
-
             const userApi = res.body.endpoints;
-        
+
             expect(userApi).toEqual(exampleApi);
         });
     })
     
 });
 
+describe('GET /api/articles', () => {
+    test('should respond with an articles array of article objects ', () => {
+        return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then((res) => {
+            const articles = res.body.articles;
+            console.log(res.body, "res.body");
+            expect(Array.isArray(articles)).toBe(true);
+            expect(articles).toHaveLength(13);
+
+            articles.forEach((article) => {
+                expect(article).toHaveProperty("author");
+                expect(article).toHaveProperty("title");
+                expect(article).toHaveProperty("article_id");
+                expect(article).toHaveProperty("topic");
+                expect(article).toHaveProperty("created_at");
+                expect(article).toHaveProperty("votes");
+                expect(article).toHaveProperty("article_img_url");
+                expect(article).toHaveProperty("comment_count");
+            })
+        });
+    });
+    test('should be sorted by date in descending order', () => {
+        return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then((res) => {
+            const articles = res.body.articles;
+            console.log(articles);
+            expect(articles).toBeSortedBy('created_at')
```

Comment:
Have a look at the docs for toBeSortedBy, I think created_at could be sorted by ascending by default doing it like this (....I think😅)

Reply:
Just realised I misread what order to put them in 😅, at first I put them In order of creation but I've amended it now thanks for spotting that